Library/Homebrew/utils/git.rb

```diff
@@ -4,7 +4,7 @@ module Git
   module_function
 
   def last_revision_commit_of_file(repo, file, before_commit: nil)
-    args = [before_commit.nil? ? "--skip=1" : before_commit.split("..").first]
+    args = ["--skip=#{before_commit.nil? ? 1 : before_commit.split("..").first}"]
```

Comment:
No, it should either be `"--skip=1"`, or when there is a commit range, i. e. `before_commit = "aabbcc...ddeeff"`, it should be just `"aabbcc"`, not `"--skip=aabbcc"`.